### ISSUE TITLE
DB Browser crashes when unticking "PK" option in the GUI table creator

### ISSUE THREAD

**Issue by Quaniful** [sqlitebrowser/sqlitebrowser#2670]
Details for the issue
--------------------

#### What did you do?
After launching DB Browser, I created a new database using the "New database" button, then pressed "Create table" and "Add".

I started adding fields to the table using the "Add" button, and chose one of the fields to be the primary key by ticking the "PK" option.

When I wanted to switch the primary key, unticking the "PK" option caused DB Browser to crash, I went through the same steps again and found that every time the "PK" option gets unticked, the whole app crashes and you have to start the process of creating a database and a table from the beginning.

#### What did you expect to see?
The column should not be primary key anymore and it's possible now to select another column as primary key.

#### What did you see instead?
The app just crashes completely.

Useful extra information
-------------------------
![db_browser_issue](https://user-images.githubusercontent.com/64565195/115359881-c58f5180-a1c7-11eb-9423-dff49d1271b1.png)

The info below often helps, please fill it out if you're able to. :)

#### What operating system are you using?

- [x] Linux: Ubuntu 20.04

#### What is your DB4S version?

- [x] Other: 3.12.99

#### Did you also

- [ ] Try out the latest nightly version: https://github.com/sqlitebrowser/sqlitebrowser#nightly-builds
- [x] Search for an existing similar issue: https://github.com/sqlitebrowser/sqlitebrowser/issues?utf8=%E2%9C%93&q=is%3Aissue%20

**Comment by chrisjlocke** [comment]
v3.12.99 indicates a nightly version. Do you know the actual build date? This should be in the help about dialog. This issue seems familiar but also quite old. Just wondering what version you're really using.

**Comment by chrisjlocke** [comment]
No, ignore me - just tried this again, and yes, it does it in the nightlies.  😮😯

**Comment by chrisjlocke** [comment]
https://user-images.githubusercontent.com/7872218/115411030-0b114600-a1eb-11eb-8994-c52a0961c97c.mp4

**Comment by MKleusberg** [comment]
Thanks, @oriesh! The crash should be fixed in the next build. Are you using the AppImage or some PPA and can install the next version to check if it's indeed working for you? :smile:

**Comment by oriesh** [comment]
I've installed it using the PPA:
```
sudo add-apt-repository -y ppa:linuxgndu/sqlitebrowser
sudo apt-get update
sudo apt-get install sqlitebrowser
```

I've formatted my system, if I reinstall it now through the PPA it'd be the updated version or do I need to install the nightly version?

Thx!

**Comment by MKleusberg** [comment]
It looks like that PPA wouldn't include the latests fixes. But here's its sibling which does:
```
sudo add-apt-repository ppa:linuxgndu/sqlitebrowser-testing
sudo apt-get update
sudo apt-get install sqlitebrowser
```
